src/api/app/models/project.rb

```diff
@@ -90,6 +90,7 @@ def staging_history
   has_many :notifications, through: :notified_projects
   has_many :reports, as: :reportable, dependent: :nullify
   has_many :label_templates
+  has_many :label_globals
```

Comment:
Done in https://github.com/openSUSE/open-build-service/pull/17275 just here until merged